### PR TITLE
Update English copy: core

### PR DESCRIPTION
**Pull request by angela-tran**
Related to #522. Covers `core:index` and `core:agency_index`

### CODE REVIEW

**Review by machikoyasuda — DISMISSED** [review]
image
image

File: benefits/locale/en/LC_MESSAGES/django.po
Comment (thekaveman) [inline benefits/locale/en/LC_MESSAGES/django.po:269]:
```suggestion
msgstr "The Cal-ITP Benefits program is currently only open to those <strong>ages 65 or older</strong>."
```

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Review by thekaveman — COMMENTED** [review]
I like how you refactored the `prefix` out.

Now that I'm seeing our new snappy page titles, I'm thinking it makes sense to reformat like:

`<Page Title> | Cal-ITP Transit Benefits`

Thoughts?

File: benefits/locale/es/LC_MESSAGES/django.po
Comment (thekaveman) [inline benefits/locale/es/LC_MESSAGES/django.po:254]:
```suggestion
msgstr "TODO: Introduction"
```

**Review by thekaveman — DISMISSED** [review]
👍

**Comment by angela-tran** [comment]
@thekaveman Looks like I pushed another commit right as you approved, sorry!

**Review by thekaveman — APPROVED** [review]

**Comment by angela-tran** [comment]
> I like how you refactored the `prefix` out.
> 
> Now that I'm seeing our new snappy page titles, I'm thinking it makes sense to reformat like:
> 
> `<Page Title> | Cal-ITP Transit Benefits`
> 
> Thoughts?

I like that better too, but I'm gathering that we don't want it to block this PR

**Comment by thekaveman** [comment]
Yeah I think we can worry about it later / next week.

**Pull request merged**
